cvat/schema.yml

```diff
@@ -6540,6 +6560,98 @@ components:
       description: |-
         * `random_uniform` - RANDOM_UNIFORM
         * `manual` - MANUAL
+    FunctionCall:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Request id
+        function:
+          $ref: '#/components/schemas/FunctionCallParams'
+        status:
+          type: string
+        progress:
+          type: integer
+          default: 0
+        enqueued:
+          type: string
+          format: date-time
+        started:
+          type: string
+          format: date-time
+        ended:
+          type: string
+          format: date-time
+        exc_info:
+          type: string
```

Comment:
I think these should be marked nullable (I see that they are set to null in the response to the initial create request).

Reply:
I couldn't find where it happens for `exc_info`, but there are other fields that can potentially be null, so I marked them.